src/index.html

```diff
@@ -1,13 +1,81 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+
+    <div
+      data-qa="card"
+      class="product-card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="Product card image"
+        class="product-card__image"
+      >
+
+      <h2
+        class="product-card__title"
+      >
```

Comment:
Переносити атрибути на нову строку потрібно тільки якщо у тебе більше двох атрибутів. Виправи всюди
```suggestion
      <h2 class="product-card__title" >
```